README.md

```diff
@@ -89,3 +95,15 @@ See [the documentation on how to run the Aeternity node if you prefer Docker][do
 # Additional resources
 
 * [Threat Model](https://github.com/aeternity/aetmodel/blob/master/ThreatModel.md)
+
+
+[circleci]: https://circleci.com/gh/aeternity/aeternity
+[circleci badge]: https://circleci.com/gh/aeternity/aeternity.svg?style=shield
+[appveyor]: https://ci.appveyor.com/project/aeternity/aeternity
+[appveyor badge]: https://ci.appveyor.com/api/projects/status/github/aeternity/aeternity?branch=master&svg=true
+[license badge]: https://img.shields.io/badge/license-ISC-blue.svg
+[license]: https://github.com/aeternity/aeternity/blob/master/LICENSE
+[erlang version badge]: https://img.shields.io/badge/erlang-20.1--21.3-blue.svg
```

Comment:
May I suggest deleting this line? It seems duplication, prone to become out of sync.
```suggestion
[erlang version badge]: https://img.shields.io/badge/erlang-20.1--20.3-blue.svg
```

Reply:
Good point.